llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -6576,29 +6576,31 @@ static bool arebothOperandsNotNan(SDValue Operand1, SDValue Operand2,
 }
 
 // FIXME: use FMINIMUMNUM if possible, such as for RISC-V.
-static unsigned getMinMaxOpcodeForFP(SDValue Operand1, SDValue Operand2,
-                                     ISD::CondCode CC, unsigned OrAndOpcode,
-                                     SelectionDAG &DAG,
-                                     bool isFMAXNUMFMINNUM_IEEE,
-                                     bool isFMAXNUMFMINNUM) {
+static unsigned
+getMinMaxOpcodeForFP(SDValue Operand1, SDValue Operand2,
+                     SDNodeFlags LHSSetCCFlags, SDNodeFlags RHSSetCCFlags,
```

Comment:
Don't really need both flag parameters if you only use LHSSetCCFlags.hasNoNaNs() && RHSSetCCFlags.hasNoNaNs()